pyani_plus/db_orm.py

```diff
@@ -245,6 +246,53 @@ def __repr__(self) -> str:
         )
 
 
+class Run(Base):
+    """Describes a single pyANI-plus run.
+
+    Each run is a set of all-vs-all comparisons between a set of FASTA
```

Comment:
If any comparison has been carried out before (with the same software/parameters) we'll be skipping it - but we'll be linking the result (`comparison`) and genomes (`genomes`) to this run as well.